index.html

```diff
@@ -37,12 +37,13 @@ <h1>Hey there. I'm Hiwot<span class="h1color"><br>
         </div>
         <div>
         <ul class="socialt">
-            <li>  <img src="image/twitter.svg" /> &nbsp;</li>
-            <li> <img src="image/vector.png" /> </li>
-            <li> <img src="image/vector (1).png" />&nbsp; </li>
-            <li> <img src="image/medium.svg" /> </li>&nbsp;
-            <li> <img src="image/github.svg" /> &nbsp;</l>
-            <li> <img src="image/angellist.svg" /> &nbsp;</l>   
+            <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp;
+                  <a href="#"><img src="image/vector.png" /></a> 
+                  <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp;
+                  <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp;
+                  <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp;
+                  <a href="#"><img src="image/angellist.svg" /></a> &nbsp;
```

Comment:
1. Good job! You're very close to the final result, but as it's mentioned by the previous reviewer you have to add the anchor (`a`) tags to your list, but you mistakenly removed the `li` tags, so please add them back because you need to wrap your anchor (`a`) tags with the `li` tags.

example :
`<li> <a href='#' > <img src="image/vector.png" /> </a> </li>`